src/index.js

```diff
@@ -66,3 +66,9 @@ exports.ui = (rootUrl, path) => {
     `https://tools.taskcluster.net/${path}` :
     `${rootUrl}/${path}`;
 };
+
+/**
+ * Return the standardized taskcluster "testing" rootUrl.
+ * Useful for nock and such things.
+ */
+exports.testRootUrl = () => 'https://tc-tests.localhost';
```

Comment:
Why is this useful to expose from source?

Reply:
No strong reason, but it helps to define this as the "official" testing rootUrl.  Historically we've made up new testing conventions for each app, and that's caused a lot of confusion.  This should help us be consistent.

Reply:
OK, as long as it's being used by more than the tests directory, I think that makes sense.